surf.deckr.deckr.yaml

```diff
@@ -0,0 +1,203 @@
+app-id: surf.deckr.deckr
+runtime: org.gnome.Platform
+runtime-version: "45"
```

Comment:
```suggestion
runtime-version: "46"
```

Reply:
You guys really hide these version numbers.  I looked for about an hour "latest runtime version" nothing comes up.  Couldn't you make this more obvious?

Reply:
It follows the latest GNOME major version and is announced on discourse. https://discourse.gnome.org/tag/announcement